OpenRA.Game/GameRules/SoundInfo.cs

```diff
@@ -48,10 +48,10 @@ public SoundInfo(MiniYaml y)
 				if (volumeModifierNode != null)
 					volumeModifier = FieldLoader.GetValue<float>(volumeModifierNode.Key, volumeModifierNode.Value.Value);
 
-				var allowInterrupt = false;
-				var allowInterruptNode = t.Value.Nodes.FirstOrDefault(x => x.Key == "AllowInterrupt");
+				var allowInterrupt = SoundPool.DefaultInterruptType;
+				var allowInterruptNode = t.Value.Nodes.FirstOrDefault(x => x.Key == "InteruptType");
```

Comment:
```suggestion
				var allowInterruptNode = t.Value.Nodes.FirstOrDefault(x => x.Key == "InterruptType");
```
Can we use `nameof(SoundPool.InterruptType)` here?

Reply:
sure